site/docs/v1/tech/account-management/index.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: doc
+title: Account Management
+description: Managing Accounts - with Self Service
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Overview
+
+FusionAuth self-service account management allows your users to manage their accounts, including the ability to add Multi-factor Authentication (MFA).  This documentation section covers adding MFA, customizing the account management user page, and troubleshooting guidance.
+
+== Setup MFA Through Self Service Page
+
+To view the account and self service pages, you will want to navigate to `icon:cube[type=fas] Applications` and click on the icon:search[role=ui-button green,type=fas]. From here copy and paste the account URL into a browser.
+
+image::account-management/applications-home.png[Home Screen, width=1200]
+
+=== Account Home
+
+Before accessing your account you will be asked to log in.
+
+image::account-management/account-login.png[Login Screen, width=1200, role=bottom-cropped]
+
+==== Account Management Home
+
+image::account-management/account-home-crop.png[Account Home, width=1200, role=bottom-cropped]
+
+From here click on `Manage Two Factor icon:arrow-right[]`
+
+On this screen you can add Two Factor methods.
+
+image::account-management/2fa-home-account-crop.png[Account Home, width=1200, role=bottom-cropped]
+
+Current acceptable types are:
+
+- TOTP / Authenticator Apps
+- SMS
+- Email
+
+If you are not seeing this screen, you may have additional steps to configure.
```

Comment:
"you may have additional steps to configure." -> "you may need additional configuration."